src/YieldDistributor.sol

```diff
@@ -51,6 +51,8 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
     uint256 public yieldFixedSplitDivisor;
     /// @notice The address of the `ButteredBread` token contract
     ERC20VotesUpgradeable public BUTTERED_BREAD;
+    /// @notice The block number before the last yield distribution
+    uint256 public beforeLastClaimedBlockNumber;
```

Comment:
maybe `previousCycleStartingBlock` would be easier to understand? like I get what it is, but the name is a bit confusing to me

Reply:
good suggestion!

Reply:
addressed with 6a814e4